templates/base.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
```

Comment:
I believe we should use 2-space tabulation in HTML files.
https://github.com/mate-academy/py-task-guideline/blob/main/html_settings/README.MD